addons/survey/static/src/js/survey_session_manage.js

```diff
@@ -6,8 +6,17 @@ var SurveyPreloadImageMixin = require('survey.preload_image_mixin');
 var SurveySessionChart = require('survey.session_chart');
 var SurveySessionTextAnswers = require('survey.session_text_answers');
 var SurveySessionLeaderBoard = require('survey.session_leaderboard');
-var core = require('web.core');
-var _t = core._t;
+const { _t } = require('web.core');
+
+const nextPageTooltips = {
+    startScreen: _t('Start'),
```

Comment:
Always unsure with those ... shouldn't we use ``_lt`` (lazy translate) here ? Otherwise I fear this is translated at file loading and not dynamically ?